srcpkgs/cross-arm-none-eabi-binutils/template

```diff
@@ -30,7 +30,7 @@ maintainer="Ivan Sokolov <ivan-p-sokolov@ya.ru>"
 license="GPL-3.0-or-later"
 homepage="https://www.gnu.org/software/${_pkgname}/"
 distfiles="${GNU_SITE}/${_pkgname}/${_pkgname}-${version}.tar.xz"
-checksum=0ab6c55dd86a92ed561972ba15b9b70a8b9f75557f896446c82e8b36e473ee04
+checksum=1b11659fb49e20e18db460d44485f09442c8c56d5df165de9461eb09c8302f85
 nocross=yes
```

Comment:
Random drive-by query - why is this nocross=yes (cross-arm-none-eabi-gdb/newlib don't have this)?